src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitxonline.QA.yaml

```diff
@@ -12,7 +12,6 @@ config:
   - ["course_experience.relative_dates", "--create", "--superusers", "--everyone",
     "--staff", "--authenticated"]
   - ["discussions.pages_and_resources_mfe", "--create", "--superusers", "--staff"]
-  - ["discussions.enable_discussions_mfe", "--create", "--everyone"]
```

Comment:
We can delete this, and manually disable the flag in the admin, or we can change the `--create` to `--disable` if we want to ensure that the flag is not enabled.

Reply:
I've already disabled it through Django Admin https://courses-qa.mitxonline.mit.edu/admin/waffle/flag/. Also checked on Prod and the flag doesn't exist there https://courses.mitxonline.mit.edu/admin/waffle/flag/. So we should be good.